src/gen/hero.h

```diff
@@ -1,18 +1,31 @@
-#ifdef _MSC_VER
-#pragma pack (1)
+#ifndef HERO_H
+#define HERO_H
+
+#if defined(__STDC_VERSION__) && __STDC_VERSION__ >= 201112L
+  #include <assert.h>
+#define STATIC_ASSERT(expr, msg) static_assert(expr, #msg)
+#else
+  #define STATIC_ASSERT(expr, msg) typedef char static_assert_##msg[(expr) ? 1 : -1]
+#endif
+
+#if defined(__BORLANDC__) || defined(__WATCOMC__)
+typedef signed long int32_t;
+#else
+typedef signed int int32_t;
 #endif
 
+STATIC_ASSERT(sizeof(int32_t) == 4, int32_needs_to_be_3_bytes);
```

Comment:
4?

Reply:
isn't 4 correct? or do you mean int32_needs_to_be_"3"_bytes